applications/browser/package.json

```diff
@@ -0,0 +1,123 @@
+{
+  "private": true,
+  "name": "theia-blueprint-browser",
+  "description": "Eclipse Theia blueprint browser product",
+  "productName": "Theia Blueprint",
+  "version": "1.35.0",
+  "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
+  "author": "Eclipse Theia <theia-dev@eclipse.org>",
+  "homepage": "https://github.com/eclipse-theia/theia-blueprint#readme",
+  "bugs": {
+    "url": "https://github.com/eclipse-theia/theia/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eclipse-theia/theia-blueprint.git"
+  },
+  "engines": {
+    "yarn": ">=1.7.0 <2",
+    "node": ">=12.14.1"
+  },
+  "theia": {
+    "frontend": {
+      "config": {
+        "applicationName": "Theia Blueprint",
+        "preferences": {
+          "toolbar.showToolbar": true,
+          "files.enableTrash": false
+        }
+      }
+    },
+    "backend": {
+      "config": {
+        "startupTimeout": -1,
+        "resolveSystemPlugins": false
+      }
+    },
+    "generator": {
+      "config": {
+        "preloadTemplate": "./resources/preload.html"
+      }
+    }
+  },
+  "dependencies": {
+    "@theia/bulk-edit": "1.35.0",
+    "@theia/callhierarchy": "1.35.0",
+    "@theia/console": "1.35.0",
+    "@theia/core": "1.35.0",
+    "@theia/debug": "1.35.0",
+    "@theia/editor": "1.35.0",
+    "@theia/editor-preview": "1.35.0",
+    "@theia/external-terminal": "1.35.0",
+    "@theia/file-search": "1.35.0",
+    "@theia/filesystem": "1.35.0",
+    "@theia/getting-started": "1.35.0",
+    "@theia/git": "1.35.0",
+    "@theia/keymaps": "1.35.0",
+    "@theia/markers": "1.35.0",
+    "@theia/messages": "1.35.0",
+    "@theia/memory-inspector": "1.35.0",
+    "@theia/metrics": "1.35.0",
+    "@theia/mini-browser": "1.35.0",
+    "@theia/monaco": "1.35.0",
+    "@theia/navigator": "1.35.0",
+    "@theia/outline-view": "1.35.0",
+    "@theia/output": "1.35.0",
+    "@theia/plugin-dev": "1.35.0",
+    "@theia/plugin-ext": "1.35.0",
+    "@theia/plugin-ext-vscode": "1.35.0",
+    "@theia/preferences": "1.35.0",
+    "@theia/process": "1.35.0",
+    "@theia/property-view": "1.35.0",
+    "@theia/scm": "1.35.0",
+    "@theia/scm-extra": "1.35.0",
+    "@theia/search-in-workspace": "1.35.0",
+    "@theia/task": "1.35.0",
+    "@theia/terminal": "1.35.0",
+    "@theia/timeline": "1.35.0",
+    "@theia/toolbar": "1.35.0",
+    "@theia/typehierarchy": "1.35.0",
+    "@theia/userstorage": "1.35.0",
+    "@theia/variable-resolver": "1.35.0",
+    "@theia/vsx-registry": "1.35.0",
+    "@theia/workspace": "1.35.0",
+    "fs-extra": "^9.0.1",
+    "theia-blueprint-updater": "1.35.0",
```

Comment:
Missed this during my last review, sorry. I think we don't need the updater for the browser app, right?

Reply:
Thanks, I've removed it